lib/Gocdb_Services/Downtime.php

```diff
@@ -738,62 +738,39 @@ public function getDowntimesFilterByParams($filterParams){
         $downtimes = $getDowntime->executeQuery();
         return $downtimes;
     }
-
     /**
-     *
-     * @param \Date $windowStart
-     * @param \Date $windowEnd
      */
-    public function getActiveAndImminentDowntimes($windowStart, $windowEnd) {
```

Comment:
so `getActiveAndImminentDowntimes` was never used it seems...

Reply:
It seems ...